### PR TITLE
Switched to jsonnet for managing instances plugins

### PR DESCRIPTION
**Pull request by mbarbero**

### CODE REVIEW

**Review by mbarbero — COMMENTED** [review]
I've reviewed all files once more, and it all LGTM. @fredg02, could you please review? Thanks.

File: instances/ee4j.ca/config.jsonnet [hunk 1]
```diff
@@ -3,7 +3,9 @@
     fullName: "ee4j.ca",
     displayName: "Jakarta Annotations"
   },
-  maven+: {
-    showVersion: false,
-  }
```
Comment (fredg02) [inline instances/ee4j.ca/config.jsonnet:8]: Is the Maven -> showVersion option still necessary?

Comment (mbarbero) [reply instances/ee4j.ca/config.jsonnet:8]: it's handled in the general config globally for all ee4j projects (see https://github.com/eclipse-cbi/jiro/blob/master/templates/config.libsonnet#L125)

Comment (fredg02) [reply instances/ee4j.ca/config.jsonnet:8]: :+1:

**Review by fredg02 — APPROVED** [review]

**Pull request merged**